flight-website/app/styles/app.css

```diff
@@ -76,6 +76,24 @@ main.ds-main {
   font-bold;
 }
 
+.ds-beta-banner {
+  display: block;
+  max-width: fit-content;
+  margin: 1.25rem auto;
+  padding: 0.75rem 1.25rem;
+  color: #911ced;
+  font-family: var(--font-body);
+  font-size: 1rem;
+  line-height: 1.2;
+  background-color: #f9f2ff;
+  border-radius: 0.25rem;
+}
+
+.ds-beta-banner a {
+  color: inherit;
+  text-decoration: underline;
+}
```

Comment:
Comment: I would nest the styles for anchor tags inside the parent selector but don't know if you prefer not to.

Reply:
This is a CSS file, not a Sass one (I was confused too, at the beginning 🙂)